distributed/client.py

```diff
@@ -343,7 +343,7 @@ class Client(object):
     distributed.scheduler.Scheduler: Internal scheduler
     """
     def __init__(self, address=None, start=True, loop=None, timeout=3,
-                 set_as_default=True, scheduler_file=None):
+                 set_as_default=True, scheduler_file=None, connection_kwargs=None):
```

Comment:
There's a misunderstanding :-) I am not asking to depend on `config.yaml`, I am asking (or suggesting) for `--certfile` and `--keyfile` to set a key in the global `config` dictionary instead of being passed as additional options to many internal APIs.